rosidl_default_generators/package.xml

```diff
@@ -19,6 +19,7 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_cpp</buildtool_export_depend>
 
+  <group_depend>rosidl_generator_packages</group_depend>
```

Comment:
do we still need to explicitly list all generators as build_export_depend here ? or do we keep it just to enforce that at least these 3 are present?

Reply:
They need to be explicit for packaging into Debians.

Reply:
:+1: yeah that was my understanding.

+1 to merge it as is.

We will revisit after the release where the dependencies for packaging should be listed as we have a conflicting policy at the moment (comparing to ros1_bridge for example)

Reply:
This PR aims to ease integrating custom generators. Making currently required generators optional would be a different thing.